roles/edpm_multipathd/tasks/configure.yml

```diff
@@ -0,0 +1,114 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Run multipathd install tasks with root privileges
+  become: true
+  block:
+    - name: Record the current multipath configuration
+      stat:
+        path: /etc/multipath.conf
+      register: stat_before
+      check_mode: false
+
+    - name: Install custom multipath.conf if one is specified
+      ansible.builtin.copy:
+        src: "{{ edpm_multipathd_custom_config_dir }}/{{ edpm_multipathd_custom_config_file }}"
+        dest: /etc/multipath.conf
+        mode: 0644
+      when:
+        - edpm_multipathd_custom_config_file|length > 0
+
+    - name: Ensure /etc/multipath.conf exists
+      when:
+        - edpm_multipathd_custom_config_file|length == 0
+      block:
+        - name: Check for existing /etc/multipath.conf
+          ansible.builtin.stat:
+            path: /etc/multipath.conf
+          register: result
+          check_mode: false
+        - name: Create /etc/multipath.conf if file is missing
+          ansible.builtin.copy:
+            src: "{{ role_path }}/files/multipath.conf"
+            dest: /etc/multipath.conf
+            mode: 0644
```

Comment:
quote `mode` as stated in the module doc: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode